pkg/system/lstat_windows.go

```diff
@@ -12,3 +26,223 @@ func Lstat(path string) (*StatT, error) {
 
 	return fromStatT(&fi)
 }
+
+func (fi *fileInfo) Size() int64 {
+	return fi.data.Size()
+}
+
+func (fi *fileInfo) Mode() (m os.FileMode) {
+	m = fi.data.Mode()
+
+	if fi.maskReparse {
+		m &^= os.ModeSymlink
+	}
+
+	return m
+}
+
+func (fi *fileInfo) ModTime() time.Time {
+	return fi.data.ModTime()
+}
+
+// Sys returns syscall.Win32FileAttributeData for file fs.
+func (fi *fileInfo) Sys() interface{} {
+	return fi.data.Sys()
+}
+
+func shouldMaskReparsePoint(name string, namep *uint16) (bool, error) {
+	var fd syscall.Win32finddata
```

Comment:
This appears to be windows-specific; perhaps it makes sense to use ""golang.org/x/sys/windows" package instead? Looks cleaner that way, and I think Go's long term plan is to move platform-specific stuff to those packages.